packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -757,9 +757,13 @@ export class DocumentDeltaConnection
 	/**
 	 * Error raising for socket.io issues
 	 */
-	protected createErrorObject(handler: string, error?: any, canRetry = true): IAnyDriverError {
+	protected createErrorObject(
+		socketEvent: string,
```

Comment:
This does not look correct by looking at the usage of this function. We are not just passing socketEvents here but also other things. We should keep the handler here or maybe replace handler with "message" if that is clearer. 

We can add another optional argument to this function with name "socketEvent" here. You can have the type string, or you can have const object with different socket events as members. As currently we are using only 1 value currently, you can just have string for now as type.